### ISSUE TITLE
Outdated project template for creating a new Roslyn analyzer?

### ISSUE THREAD

**Issue by robkroll** [dotnet/roslyn-sdk#548]
I am trying to follow the steps on this blog post: https://devblogs.microsoft.com/dotnet/how-to-write-a-roslyn-analyzer/

I have updated to VS 16.6.2 and installed the .NET Compiler Platform SDK.  I created a project with "Analyzer with Code Fix (.NET Standard)" project template, but it doesn't compile. I am getting at least a couple of issues:

* A package source with the name "dotnet.myget.org roslyn" and package source https://dotnet.myget.org/F/roslyn/api/v3/index.json were added when the .NET Compiler Platform SDK was installed. The test project that is created references `Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.MSTest` at version "1.0.1-beta1.0". The package is not available at that source, but _is_ at  [roslyn-analyzers](https://dotnet.myget.org/feed/roslyn-analyzers/package/nuget/Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.MSTest), but not for this version. When I add the roslyn-analyzers package source and use the latest version, it still doesn't fix the errors in the created unit test file:
```
using Verify = Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.MSTest.CodeFixVerifier<CodeGenAnalyzerAnalyzer, CodeGenAnalyzerCodeFixProvider>;
```
When the package is added, `CodeFix` is still not a found namespace.

* The VSIX project that is created has the same Assembly as the project containing the analyzer. It fails to restore the NuGet packages and build. 

* When I did manage to build it (by unloading the test project and changing the VSIX project assembly name) the Experimental VS instance listed the extension in the Extension Manager but I couldn't hit a breakpoint on the analyzer project.

Am I just using an out of date template to create the project? Should I downgrade to an older version of VS? I had hoped this would 'just work' a little bit more. Any help is appreciated.

**Comment by sharwell** [comment]
Duplicate of #514

**Comment by sharwell** [comment]
These issues should be fixed for the 16.7 release 👍
